src/app/component/Field/Field.component.js

```diff
@@ -146,6 +149,60 @@ class Field extends Component {
         }
     }
 
+    handleSelectListKeyPress(event) {
+        const {
+            isSelectExpanded,
+            searchString: prevSearchString,
+            valueIndex: previousValueIndex
+        } = this.state;
+        const keyCode = event.which || event.keycode;
+
+        // on Enter pressed
+        if (keyCode === 13) {
+            this.handleSelectExpand();
+            return;
+        }
+
+        if (!isSelectExpanded
+            || !keyCode
+            || keyCode < 65
+            || keyCode > 122
+            || (keyCode > 90 && keyCode < 97)
+        ) return;
+
+        const { selectOptions, onChange, id: selectId } = this.props;
+        const pressedKeyValue = String.fromCharCode(keyCode).toLowerCase();
+
+        let searchString = (prevSearchString[prevSearchString.length - 1].toLowerCase() !== pressedKeyValue)
+            ? `${prevSearchString}${pressedKeyValue}`
+            : pressedKeyValue.toUpperCase();
+
+        let selectedItemIndex = selectOptions.findIndex(({ label }, i) => (
+            label && label.startsWith(searchString) && (
```

Comment:
You should better make label lowercase, rather then lowercasing / uppercasing search string.